### PR TITLE
Bump freetype to 2.8

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/library/freetype/Makefile [hunk 1]
```diff
@@ -49,6 +48,10 @@ CONFIGURE_OPTIONS+= --with-png
 CONFIGURE_OPTIONS+= --with-harfbuzz
 CONFIGURE_OPTIONS+= --with-pic
 
+COMPONENT_PREP_ACTION = ( cd $(@D); \
+	$(GSED) -ri 's:.*(AUX_MODULES.*valid):\1:' modules.cfg && \
+	$(GSED) -r 's:.*(\#.*SUBPIXEL_RENDERING) .*:\1:' \
+		-i include/freetype/config/ftoption.h )
```
Comment (pyhalov) [inline components/library/freetype/Makefile:54]: Will it affect existing applications or behavior?

Comment (alarcher) [reply components/library/freetype/Makefile:54]: It enables better hinting of fonts. In any case the update of freetype itself, without this, affects how fonts are rendered.

Comment (alarcher) [reply components/library/freetype/Makefile:54]: The subsequent fontconfig PR defines a configuration to make the default rendering look nicer.

Comment (pyhalov) [reply components/library/freetype/Makefile:54]: should applications, using freetype, be recompiled?

Comment (alarcher) [reply components/library/freetype/Makefile:54]: No it is not required. We may want to recompile cairo to support the additional hinting at some point, but that's all.

**Pull request merged**